lib/Environment.js

```diff
@@ -50,6 +52,28 @@ Environment.prototype = {
      */
     waitForPage: function (timeout) {
         throw new Error('You must redefine waitForPage method in child environment');
+    },
+
+    back: function () {
```

Comment:
Why do we need to add those functions to Env?
Before it was enough to add it in Actions.